chemicaltoolbox/autodock_vina/docking/docking.xml

```diff
@@ -1,4 +1,4 @@
-<tool id="docking" name="Docking" version="0.2.1">
+<tool id="docking030" name="Docking030" version="0.3.0">
```

Comment:
the ID needs to stay the same, otherwise Galaxy does not know that it is the same tool

Reply:
I would also keep the name

Reply:
I forgot to revert the names - now they are back to the originals.